HighAvailabilityModule.UnitTest/InMemoryMembershipServerTest.cs

```diff
@@ -16,11 +17,27 @@ public class InMemoryMembershipServerTest
         private static DateTime Now { get; } = DateTime.Parse("2019-09-27T12:00:00.2965246Z");
 
         private static string Client1Uuid => "cdca5b45-6ea1-4d91-81f6-d39f4821e791";
+        private static string ClientUtypeA => "A";
+        private static string ClientUnum1 => "1";
 
-        private static string Client2Uuid => "33253ab7-27b6-478c-a359-4eca7df83b80";
+        private static string Client2Uuid => "39a78df0-e101-49b9-8c56-ec2fea2e47df";
+        private static string ClientUnum2 => "2";
+
+        private static string Client3Uuid => "33253ab7-27b6-478c-a359-4eca7df83b80";
+        private static string ClientUtypeB => "B";
 
         private InMemoryMembershipServer server;
 
+        private void TestFunc(Dictionary<string, HeartBeatEntry> Current, string Uuid, string Utype, string Unum)
```

Comment:
This is duplicated code, can we consolidate them? If it is not possible, we shall also change the func name.